platform/keysystem/keyserver/keyserver.go

```diff
@@ -16,6 +17,12 @@ func main() {
 	if err != nil {
 		logger.Fatal(err)
 	} else {
```

Comment:
Let's remove the unnecessary `else` clauses after `if ... { logger.Fatal() }` to avoid arbitrarily deep nesting

Reply:
I have removed the unnecessary else clause. I suppose I didn't realize that `logger.Fatal` actually calls exit, hence why I wrote it that way.